engine/src/exec-multimedia.cpp

```diff
@@ -356,7 +367,17 @@ void MCMultimediaExecRecord(MCExecContext& ctxt, MCStringRef p_filename)
         MCPlatformSoundRecorderStart(MCrecorder, *soundfile);
 #else
 	extern void MCQTRecordSound(MCStringRef soundfile);
-	MCQTRecordSound(*soundfile);
+	
+	// PM-2015-08-04: [[ Bug 15321 ]] Make sure we always pass an absolute path to MCQTRecordSound
+	MCStringRef t_absolute_path;
+	if (!is_windows_absolute_path(*soundfile))
+		ctxt . GetObject()->getstack()->resolve_relative_path(*soundfile, t_absolute_path);
```

Comment:
I'm not sure this should be resolving relative to the current script's owning stack. It should just be resolving relative as normal - i.e. if relative path, prepend CWD; otherwise leave as is. i.e MCS_get_canonical_path()?